app/src/features/charts/DisparityBarChart.tsx

```diff
@@ -211,19 +215,23 @@ function TwoVarBarChart(props: {
   breakdownVar: string;
   breakdownVarDisplayName: string;
 }) {
+  const [ref, width] = useResponsiveWidth();
   return (
-    <Vega
-      spec={getSpec(
-        props.data,
-        props.breakdownVar,
-        props.breakdownVarDisplayName,
-        props.thickMeasure,
-        props.thickMeasureDisplayName,
-        props.thinMeasure,
-        props.thinMeasureDisplayName
-      )}
-    />
+    <div ref={ref as RefObject<HTMLDivElement>}>
+      <Vega
+        spec={getSpec(
+          props.data,
+          width ? width : 100, // Set a default value until width is set
```

Comment:
Perhaps useResponsiveWidth should accept a default value on initialization

Reply:
ya good call!